SWLOR.Game.Server/Feature/PerkDefinition/OneHandedPerkDefinition.cs

```diff
@@ -82,25 +87,165 @@ private void DualWield()
                 .Name("Dual Wield")
 
                 .AddPerkLevel()
-                .Description("Enables the use of two one-handed weapons at the same time at -20%/-20% to hit. [Cross Skill]")
-                .Price(6)
+                .Description("Enables the use of two one-handed weapons at the same time at -10%/-10% to hit. [Cross Skill]")
+                .Price(4)
                 .RequirementSkill(SkillType.OneHanded, 15)
                 .GrantsFeat(FeatType.DualWield);
         }
 
+        private void WailingBlows()
+        {
+            _builder.Create(PerkCategoryType.OneHandedGeneral, PerkType.WailingBlows)
+                .Name("Wailing Blows")
+
+                .AddPerkLevel()
+                .Description("Defending against two blades opens up your opponents. While dual-wielding one-handed weapons, you gain +15% critical chance.")
```

Comment:
Let's remove the flavor text from the description. The description should really be what the functional mechanics are and I think it gets muddied with the RP info. I'm open to adding a field for flavor text as part of another update though.

To be specific I'm asking to remove this section: "Defending against two blades opens up your opponents."